app_dart/lib/src/foundation/github_checks_util.dart

```diff
@@ -40,6 +40,27 @@ class GithubChecksUtil {
     );
   }
 
+  /// Finds all check suites that are associated with a given git [ref].
```

Comment:
Streams add a lot of complexity in Dart. Simplifying this to an iterable will make it easier to use.

Reply:
Sounds good, updated